pinot-tools/src/main/java/com/linkedin/pinot/tools/segment/converter/ColumnarToStarTreeConverter.java

```diff
@@ -118,6 +120,11 @@ private void convertSegment(File columnarSegment)
     config.setStarTreeIndexSpecFile(_starTreeConfigFileName);
     config.setOverwrite(_overwrite);
 
+    // Load the StarTreeSpec object from the config file.
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
Try to define ObjectMapper as private static final member variable to reduce garbage.